console/src/components/Services/Data/RawSQL/RawSQL.js

```diff
@@ -529,6 +483,56 @@ const RawSQL = ({
   );
 };
 
+/**
+ * # Alert
+ * @component
+ *
+ * @param {*} { ongoingRequest, lastError, lastSuccess, className }
+ */
+const Alert = ({ ongoingRequest, lastError, lastSuccess, className }) => (
+  <div className="hidden col-xs-4">
+    <div className={`${className} col-xs-12`}>
+      {ongoingRequest && (
+        <div className="hidden alert alert-warning" role="alert">
+          Running...
+        </div>
+      )}
+      {lastError && (
+        <div className="hidden alert alert-danger" role="alert">
+          Error: {JSON.stringify(lastError)}
+        </div>
+      )}
+      {lastSuccess && (
+        <div className="hidden alert alert-success" role="alert">
+          Executed Query
+        </div>
+      )}
+      ;
+    </div>
+  </div>
+);
+/**
+ * #RunButton
+ * @component
+ *
+ * @param {*} { onClick, className }
+ * @returns
+ */
+const RunButton = ({ onClick, className }) => {
```

Comment:
Is a separate component needed? There is no additional logic in `RunButton`, we can just use `Button` instead. JSX will be a couple of lines bigger, but there will be fewer components.

Reply:
Yeah will change, I thought this will keep the main render method clean and much readable.